src/payment/strategies/chasepay/chasepay-payment-strategy.ts

```diff
@@ -0,0 +1,228 @@
+                    .then(({ ChasePay }) => {
+                        const { START_CHECKOUT, COMPLETE_CHECKOUT, CANCEL_CHECKOUT } = ChasePay.EventType;
+
+                        ChasePay.on(CANCEL_CHECKOUT, () => this._onCancel());
+                        ChasePay.on(COMPLETE_CHECKOUT, (payload: ChasePaySuccessPayload) => this._completeCheckout(payload, methodId)
+                            .then(() => this._onPaymentSelect())
+                            .catch(error => this._onError(error)));
+                        ChasePay.on(START_CHECKOUT, () => this._paymentInstrumentSelected(methodId));
+
+                        if (ChasePay.isChasePayUp && document.getElementById(this._container || '') && !this._isEditing) {
+                            ChasePay.insertBrandings({
+                                containers: [this._container],
+                                color: 'white',
+                            });
+                        }
+                    });
+            })
+          .then(() => super.initialize(options));
+    }
+
+    execute(orderRequest: OrderRequestBody, options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
+        const { payment, ...order } = orderRequest;
+
+        if (!payment) {
+            throw new InvalidArgumentError('Unable to proceed because the customer has not signed into their ChasePay wallet account or selected a payment option.');
+        }
+
+        if (!this._paymentMethod || !this._paymentMethod.initializationData) {
+            throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
+        }
+
+        const { initializationData } = this._paymentMethod;
+
+        if (!initializationData.paymentCryptogram || this._isEditing) {
+            return this._store.dispatch(this._orderActionCreator.submitOrder(order, options))
+                .then(() => this._displayChasepayLightbox(this._paymentMethod))
+                .catch(error => this._onError(error));
+        } else {
+            const riskToken = (payment.methodId === 'wepay') ? this._wepayRiskClient.getRiskToken() : undefined;
+            const paymentData = this._mapCardData(initializationData, riskToken);
+
+            return this._store.dispatch(this._orderActionCreator.submitOrder(order, options))
+                .then(() =>
+                    this._store.dispatch(this._paymentActionCreator.submitPayment({
+                        ...payment,
+                        paymentData,
+                    }))
+                );
+        }
+    }
+
+    private _displayChasepayLightbox(paymentMethod: PaymentMethod | undefined): any {
+        if (!paymentMethod || !paymentMethod.initializationData) {
+            throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
+        }
+        const { initializationData } = paymentMethod;
+
+        return this._chasePayScriptLoader.load(paymentMethod.config.testMode)
+            .then(({ ChasePay }) => {
```

Comment:
You need to configure `ChasePay`. I notice you've added the locale configuration in `ChasepayCustomerStrategy`. That's not enough because once the shopper has signed in, and they refresh the page, the customer strategy won't get executed anymore.

Reply:
Yeah, forgot to add the configuration in here. It's already fixed.